.circleci/config.yml

```diff
@@ -1,12 +1,23 @@
+---
+lint_and_unit: &lint_and_unit
+  - delivery
+  - danger
+  - lint-yaml
+  - lint-markdown
+
 version: 2.1
 
 orbs:
-  kitchen: sous-chefs/kitchen@2
+  kitchen: sous-chefs/kitchen@2.0.2
```

Comment:
These do the same things, `@2` should mean, use the latest of the 2 series

Reply:
Changed back to @2.